### ISSUE TITLE
ZNC Not Working with PushBullet

### ISSUE THREAD

**Issue by Hink1** [jreese/znc-push#215]
I have tried thorough debugging to try and get PushBullet to work with the Push module. I have enabled the module and set my secret, target as the [device ID](https://www.reddit.com/r/PushBullet/comments/1vof0d/ways_to_get_device_id/ceuaw3t/), and sent several test messages. None of them reached PushBullet. I followed [this](https://firrre.com/pushbullet/) guide as well. I am on ZNC 1.6.3.

**Comment by marsjaninzmarsa** [comment]
Why closed?

**Comment by Jusix** [comment]
Same problem, it worked for years and suddendly stopped....

**Comment by lrbays** [comment]
I'm having the same issue. Tried everything I could think of both on Pushbullet's side and on ZNC/ZNC Push's side, to no avail. It works fine with other services, though.

**Comment by lrbays** [comment]
Update: it works again after rebuilding with curl=yes

**Comment by Jay2k1** [comment]
thank you so much @lrbays -- that was the hint I needed. So essentially the solution is to not only download the .cpp and run znc-buildmod, as I used to do, but download the whole thing including the makefile (as you usually do), and then run ```make curl=yes``` and use the resulting .so file.

At first, even though I ran unloadmod and loadmod afterwards, it still seemed to use the old version, because it still didn't work and I could see an open file handle to the deleted .so file, but after a znc restart, it worked. Thanks again.

Although the documentation says that using curl is recommended, I still feel like this issue should be open because there's something that clearly needs to be addressed. Using default settings or by compiling with znc-buildmod, as it seems, pushbullet _will not work anymore_. cc @jreese
